python/examples/tchannel_server.py

```diff
@@ -35,8 +35,8 @@ def main():  # pragma: no cover
 
     handler = get_example_handler()
 
-    server = client.host(args.port, handler)
-    server.listen()
+    server = client.host(handler)
+    server.listen(args.port)
```

Comment:
The hostname for peers that can connect to you? It should be trivial to add that later.